### PR TITLE
    feature(.node.py): Fixing stress command for docker cluster

### PR DESCRIPTION
**Pull request by Orenef11**
* Create `docker_image` variable under `Node` class
* Change the `jmx_port` field under ScyllaDockerNode from
    int to string (because the old CCN code expected to get
    string)
* stress method - Convert the `stress` variable to list and 
    set None value for `cwd` variable in case the cluster is 
    docker

### CODE REVIEW

File: ccmlib/scylla_cluster.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def __init__(self, path, name, partitioner=None, install_dir=None,
             install_dir, self.scylla_mode = install_func(install_dir)
 
         self.started = False
-        self.force_wait_for_cluster_start = (force_wait_for_cluster_start != False)
```
Comment (fruch) [inline ccmlib/scylla_cluster.py:44]: why are you changing this ?

File: ccmlib/node.py [hunk 2]
```diff
@@ -1133,14 +1134,20 @@ def stress(self, stress_options=None, capture_output=False, **kwargs):
         else:
             stress_options = stress_options[:]
 
-        stress = common.get_stress_bin(self.get_install_dir())
+        if self.docker_name:
```
Comment (fruch) [inline ccmlib/node.py:1137]: replace with `self.is_docker()`

File: ccmlib/scylla_cluster.py [hunk 3]
```diff
@@ -28,7 +28,7 @@ def __init__(self, path, name, partitioner=None, install_dir=None,
         install_func = common.scylla_extract_install_dir_and_mode
 
         cassandra_version = kwargs.get('cassandra_version', version)
-        docker_image = kwargs.get('docker_image')
+        docker_image = kwargs.pop('docker_image', None)
```
Comment (fruch) [inline ccmlib/scylla_cluster.py:31]: why are you using pop here ?

File: ccmlib/scylla_cluster.py (hunk 1 above)
Comment (Orenef11) [reply ccmlib/scylla_cluster.py]: I forgot to revert this change.
Fixed

File: ccmlib/node.py (hunk 2 above)
Comment (Orenef11) [reply ccmlib/node.py]: Fixed

File: ccmlib/scylla_cluster.py (hunk 3 above)
Comment (Orenef11) [reply ccmlib/scylla_cluster.py:31]: I dont know there is `is_docker` method and I passed it tho all Node object (I revert this change)

Comment (fgelcer) [reply ccmlib/scylla_cluster.py:31]: i still see it is `poping` it here

**Review by fgelcer — COMMENTED** [review]
waiting for a you to replace the `kwargs.pop` to be reverted...

File: ccmlib/cluster.py [hunk 4]
```diff
@@ -43,12 +43,12 @@ def __init__(self, path, name, partitioner=None, install_dir=None, create_direct
         # backwards compatibility.
         version = kwargs.get('cassandra_version', version)
         install_dir = kwargs.get('cassandra_dir', install_dir)
-        docker_image = kwargs.get('docker_image')
+        self.docker_image = kwargs.get('docker_image')
```
Comment (fruch) [inline ccmlib/cluster.py:46]:
```suggestion
        docker_image = kwargs.get('docker_image')
```

File: ccmlib/cluster.py [hunk 5]
```diff
@@ -43,12 +43,12 @@ def __init__(self, path, name, partitioner=None, install_dir=None, create_direct
...
         if create_directory:
             # we create the dir before potentially downloading to throw an error sooner if need be
             os.mkdir(self.get_path())
 
-        if docker_image:
+        if self.docker_image:
```
Comment (fruch) [inline ccmlib/cluster.py:51]:
```suggestion
        if docker_image:
```

**Review by fruch — CHANGES_REQUESTED** [review]
don't assign `docker_image` in `cluster.py`

File: ccmlib/scylla_cluster.py (hunk 3 above)
Comment (Orenef11) [reply ccmlib/scylla_cluster.py]: Thanks, done :-)

File: ccmlib/cluster.py (hunk 4 above)
Comment (Orenef11) [reply ccmlib/cluster.py]: Thanks, done :-)

File: ccmlib/cluster.py (hunk 5 above)
Comment (Orenef11) [reply ccmlib/cluster.py]: Thanks, done :-)

**Pull request merged**